languages/javalang/builder-renderer/generated-code/src/test/java/com/commercetools/utils/CommercetoolsTestUtils.java

```diff
@@ -28,4 +32,30 @@ public static LocalizedString randomLocalizedString() {
     public static String getProjectKey() {
         return System.getenv("JVM_SDK_IT_PROJECT_KEY");
     }
+
+    public static String getClientId() {
+        return System.getenv("JVM_SDK_IT_CLIENT_ID");
+    }
+
+    public static String getClientSecret() {
+        return System.getenv("JVM_SDK_IT_CLIENT_SECRET");
+    }
+
+    public static String getScopes() {
+        return "manage_project:"+getProjectKey();
+    }
+
+    public static ApiRoot getApiRoot(){
+        return ApiRoot.formMiddlewares(
+                new CtpOAuthMiddlewareForClientCredentialsGrant(
+                        getClientId(),
+                        getClientSecret(),
+                        "manage_project:achraf-61",
```

Comment:
Hm... Here we should the correct project key...

Reply:
maybe have a environment variable for the project key ?